packages/core/src/service/helpers/settings-store/settings-store.service.ts

```
@@ -0,0 +1,455 @@
+     * @description
+     * Get the field configuration for a key.
+     */
+    getFieldDefinition(key: string): SettingsStoreFieldConfig | undefined {
+        return this.fieldRegistry.get(key);
+    }
+
+    /**
+     * @description
+     * Validate a value against its field definition.
+     */
+    async validateValue(key: string, value: any, ctx: RequestContext): Promise<string | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)